g.py

```diff
@@ -10,10 +10,10 @@
 ver='24'
 # only one 100% per game
 
-UP=(264,273)
-DOWN=(258,274)
-LEFT=(260,276)
-RIGHT=(262,275)
+UP=(264, 1073741906)
+DOWN=(258, 1073741905)
+LEFT=(260, 1073741904)
+RIGHT=(262, 1073741903)

```

Comment:
Haven't looked at the commit that introduced this, but it'll be better not to have hard coded values for things like this as they're bound to change.

Would be great to drop the hard coded values.